src/components/Nav.jsx

```diff
@@ -15,6 +15,7 @@ export default class Nav extends React.Component {
           <a href="/students">Students</a>
           <a href="/events">Events</a>
           <a href="/superlatives">Superlatives</a>
+          <a href="/superlatives">Shoutouts</a>
```

Comment:
This should be `/shoutouts` instead of `/superlatives`

Reply:
Good catch. Fixing now!